crates/reporting/src/report.rs

```diff
@@ -260,13 +260,13 @@ pub struct StyleCodes {
 }
 
 pub const ANSI_STYLE_CODES: StyleCodes = StyleCodes {
-    red: "\u{001b}[31m",
-    green: "\u{001b}[32m",
-    yellow: "\u{001b}[33m",
-    blue: "\u{001b}[34m",
-    magenta: "\u{001b}[35m",
-    cyan: "\u{001b}[36m",
-    white: "\u{001b}[37m",
+    red: "\u{001b}[38;5;9m",
+    green: "\u{001b}[38;5;10m",
+    yellow: "\u{001b}[38;5;11m",
+    blue: "\u{001b}[38;5;14m",
```

Comment:
As the light blue (#0000ff) doesn't pass [wcag AA standard](https://webaim.org/resources/contrastchecker/) I picked aqua (#00ffff) instead.